versions.md

```diff
@@ -4,6 +4,9 @@ Version 2.6.0
 * New TransportInterface and NetworkFactory API to allow updating the network interfaces at runtime (ABI breaks on RPTS
   and transport layers)
 * Removed dll export for constructors and destructors of factory created entities (breaks ABI)
+* Added methods getting `fastrtps::Time_t` as parameters instead of `fastrtps::rtps::Time_t`.
+* Changed signature of eprosima::fastdds::dds::DataWriter::dispose_w_timestamp. (ABI break).
+* Added method getting `std::vector<InstanceHandle_t>&` instead of `std::vector<InstanceHandle_t*>&`.
```

Comment:
```suggestion
* Added methods getting `fastrtps::Time_t` as parameters instead of `fastrtps::rtps::Time_t` (API extension, API deprecations).
* Changed signature of eprosima::fastdds::dds::DataWriter::dispose_w_timestamp (ABI break).
* Added method getting `std::vector<InstanceHandle_t>&` instead of `std::vector<InstanceHandle_t*>&` (API extension, API deprecations).
```